tests/Resources/modules/translationtest/vendor/composer/ClassLoader.php

```diff
@@ -279,7 +279,7 @@ public function isClassMapAuthoritative()
      */
     public function setApcuPrefix($apcuPrefix)
     {
-        $this->apcuPrefix = function_exists('apcu_fetch') && ini_get('apc.enabled') ? $apcuPrefix : null;
+        $this->apcuPrefix = function_exists('apcu_fetch') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN) ? $apcuPrefix : null;
```

Comment:
Interesting way of checking a php config value (**FILTER_VALIDATE_BOOLEAN**), never saw it before 👍

Reply:
That was composer, not me, but interesting nonetheless!